service/user/actions.go

```diff
@@ -134,7 +136,7 @@ func formatUserResponse(user datastore.User, w http.ResponseWriter) error {
 
 	// Encode the response as JSON
 	if err := json.NewEncoder(w).Encode(response); err != nil {
-		log.Println("Error forming the user response.")
+		svlog.Error("error-user-response", err)
```

Comment:
I guess the idea behind this is searching for the whole message under 'error-user-response' and printing it along with the tag, right?

Reply:
It's mainly for log analytics. It makes it simpler to do reporting on a specific type of error based on the code.

Reply:
I wonder if having an int error_code and a free error_message text could be a solution. Analytics is a M2M interface that maybe prefers an integer (just a thought)